Directory.Build.props

```diff
@@ -1,7 +1,10 @@
 <Project>
 
 	<PropertyGroup>
-    
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Unrelated improvement. This was missing in the Directory.Build.props. 
Every project had this different.